packages/language/test/linking.test.ts

```diff
@@ -274,6 +274,21 @@ describe("Linking tests", () => {
  DCL <|a:A|> CHAR(8) INIT("A");`));
   });
 
+  describe("Preprocessor", () => {
+    /**
+     * This test intentionally fails in order to test that a native
+     * PL/I variable does not link to a preprocessor variable.
+     */
+    test.fails("Must not link between PL/I and preprocessor variables", () =>
```

Comment:
Anti-case is good to avoid to avoid a regression on this issue, but we should also have another case that asserts correct linking for both the pre-processor & pli vars in the same example.